### PR TITLE
Release changes

### PR DESCRIPTION
**Pull request by avihric**
Increased version number
Contains changes from @AdrianSch in #1274

### CODE REVIEW

File: catroid/res/values-de/strings.xml [hunk 1]
```diff
@@ -462,14 +462,14 @@
   <string name="phiro_motor_stop">Stop Phiro motor</string>
   <string name="brick_phiro_motor_forward_action">Move Phiro motor forward</string>
   <string name="brick_phiro_motor_backward_action">Move Phiro motor backward</string>
```
Comment (obusher) [inline catroid/res/values-de/strings.xml]: english!

Comment (avihric) [inline catroid/res/values-de/strings.xml]: Again, not everything is translated so far in German. That's why you see english strings.

Comment (robertpainsi) [inline catroid/res/values-de/strings.xml]: ...and I thought, what's wrong with the word 'backward'? :joy_cat:

Comment (obusher) [inline catroid/res/values-de/strings.xml]: :lollipop:

**Comment by avihric** [comment]
Failed test single run:
https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/378/

**Pull request merged**